backend/utils/hf_progress.py

```
@@ -321,45 +343,84 @@ def patched_update(tqdm_self, n=1):
-                                setattr(module, attr_name, original)
-                        except (AttributeError, TypeError):
-                            pass
-                    self._patched_modules = {}
+            if cls._original_tqdm_auto:
+                tqdm_module.auto.tqdm = cls._original_tqdm_auto
 
-                    # Restore hf_tqdm's original update method
-                    if self._hf_tqdm_original_update:
-                        try:
-                            from huggingface_hub.utils import tqdm as hf_tqdm_module
+            for key, (module, attr_name, original) in cls._patched_modules.items():
```

Ruff (0.15.20)

[warning] Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)